scripts/onRelease.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * ./scripts/onRelease.js
+ *
+ * Executed during the release phase in a heroku environment.
+ */
```

Comment:
Do we need this file ? This is just one liner that's why.  can we do "on-release": "node db/migrate.js" ?